ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/ChatReply.qml

```diff
@@ -85,7 +85,7 @@ Loader {
 
             Separator {
                 anchors.top: repliedMessageType == Constants.imageType ? imgReplyImage.bottom : lblReplyMessage.bottom
-                anchors.topMargin: repliedMessageType == Constants.imageType ? 15 : 8
```

Comment:
For the record, I was wondering why it was set to `15` in case of an image reply in the first place and thought it might be needed in normal mode. However, I tested this in normal mode and even going there with `8` all the time looks better than `15`. So I decided to remove this.